app/i18n/en/gen.php

```diff
@@ -203,17 +203,18 @@
 		'mastodon' => 'Mastodon',
 		'movim' => 'Movim',
 		'pinboard' => 'Pinboard',
-		'pinterest' => 'Pinterest',	// TODO
+		'pinterest' => 'Pinterest',
 		'pocket' => 'Pocket',
 		'print' => 'Print',
 		'raindrop' => 'Raindrop.io',
-		'reddit' => 'Reddit',	// TODO
+		'reddit' => 'Reddit',
 		'shaarli' => 'Shaarli',
 		'twitter' => 'Twitter',
 		'wallabag' => 'wallabag v1',
 		'wallabagv2' => 'wallabag v2',
-		'whatsapp' => 'Whatsapp',	// TODO
-		'xing' => 'Xing',	// TODO
+		'web-sharing-api' => 'Default System Sharing',
```

Comment:
```suggestion
		'web-sharing-api' => 'Default system sharing',
```


But maybe something like?
```suggestion
		'web-sharing-api' => 'Share through system',
```

Reply:
`Share through system` does not make me more happier.

Reply:
Maybe @Alkarex has a nice idea

Reply:
I cannot find anything much better. I have proposed "Standard share" for the French version